aspnetcore/blazor/hybrid/developer-tools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Use browser developer tools with ASP.NET Core Blazor Hybrid
+author: guardrex
+description: Learn how to use browser developer tools with Blazor Hybrid apps.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: "mvc"
+ms.date: 03/23/2022
+no-loc: ["Blazor Hybrid", Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/hybrid/developer-tools
+---
+# Use browser developer tools with ASP.NET Core Blazor Hybrid
+
+This article explains how to use [browser developer tools](https://developer.mozilla.org/docs/Glossary/Developer_Tools) with Blazor Hybrid apps.
+
+[!INCLUDE[](~/blazor/includes/blazor-hybrid-preview-notice.md)]
+
+## .NET MAUI Blazor Hybrid
+
+To use developer tools with a .NET MAUI Blazor app, use the guidance in the following sections depending on the operating system used to run the app:
+
+* [iOS](#ios)
+* [Android](#android)
+
+### iOS
+
+Use the following steps to use Safari developer tools with an iOS app. The following example shows how to inspect an unhandled exception in the **Web Inspector** console log:
+
+1. Open desktop Safari.
+1. Select the **Preferences** > **Advanced** > **Show Develop** menu item.
+1. Run the .NET MAUI Blazor app in the Visual Studio emulator.
```

Comment:
> Visual Studio emulator

Not sure what this may be referring to? Perhaps:

```suggestion
1. Run the .NET MAUI Blazor app in the iOS simulator.
```